src/Manager/CampaignDraftManager.php

```diff
@@ -0,0 +1,214 @@
+<?php
+
+namespace Mailjet\MailjetBundle\Manager;
+
+use \Mailjet\Resources;
+use \Mailjet\Response;
+use Mailjet\MailjetBundle\Client\MailjetClient;
+use Mailjet\MailjetBundle\Exception\MailjetException;
+use Mailjet\MailjetBundle\Model\CampaignDraft;
+
+/**
+ * https://dev.mailjet.com/email-api/v3/campaigndraft/
+ * CampaignDraft data. (list,view, create, update, delete,schedule,send ...)
+ */
+class CampaignDraftManager {
+
+    /**
+     * Mailjet client
+     * @var MailjetClient
+     */
+    protected $mailjet;
+
+    public function __construct(MailjetClient $mailjet) {
+        $this->mailjet = $mailjet;
+    }
+
+    /**
+     * List campaigndraft resources available for this apikey
+     * @return array
+     */
+    public function getAllCampaignDrafts(array $filters = null) {
+        $response = $this->mailjet->get(Resources::$Campaigndraft, ['filters' => $filters]);
+        if (!$response->success()) {
```

Comment:
We already have a throwError method which is an helper here. I think we should let this for now and in a futur evolution think about this. Create a sort of ResponseHandler which will handle errors, paginations, data, ...